plugins/module_utils/foreman_helper.py

```diff
@@ -261,7 +273,7 @@ def _patch_subnet_rex_api(self):
             see https://projects.theforeman.org/issues/19086
         """
 
-        if 'remote_execution_features' not in self.foremanapi.apidoc['docs']['resources']:
+        if not self.has_plugin('remote_execution'):
```

Comment:
We could move this patching to the place where we check for the availability of the plugin, right? No need to patch, if it is not used...

Reply:
You mean only patch if the param is used?

Reply:
I meant only patch if the plugin is required.

Reply:
This is not going to be a blocker for me.

Reply:
I'd like to leave that out for now. Feel free to open a follow up ;)